### PR TITLE
Remove handheld crew monitor

### PR DESCRIPTION
**Pull request by metalgearsloth**
- It's supposed to be a CMO computer.
- It's getting powercrept and added everywhere.

I don't care much about the syndie but it baits people into shoving it into NT again.

Also I just found out there's massive copy-paste in head lockers so woopy.

:cl:
- remove: Remove handheld crew monitors in lieu of computers.

### CODE REVIEW

**Comment by thebadman4662** [comment]
Would it not be better to remove from tech instead and mark as objective? If its tech it makes sense paramedics every now and then get it, especially when medical can print them, albeit they cost a good chunk of materials.

**Comment by LordEclipse** [comment]
> Would it not be better to remove from tech instead and mark as objective? If its tech it makes sense paramedics every now and then get it, especially when medical can print them, albeit they cost a good chunk of materials.

Actually yeah, it could be tried as an objective, like the QM's digiboard, for instance.

**Comment by metalgearsloth** [comment]
Restricting + stealing a CM board is better as an objective than any number of pre-mapped handheld CMs.

**Comment by thebadman4662** [comment]
So, crew monitor will be only in CMO office now? I do not envy paramedics unless its going to be special kind of monitor even if functionally same to the ones you can get from cargo for 2k.

**Comment by Rainfey** [comment]
I dont believe Origin has a crew monitor computer, though there is a board that spawns in maints.

**Comment by Lomovar** [comment]
Why not leave it as a researchable thing? It would make science more useful...why do we ever remove anything instead of adding something new?

**Comment by Holinka4ever** [comment]
I guess we need a disscussion label, i mean this notable change need to be discussion before implemeting, because it will hit a lot of stones in a water:

1) Rotting round remove, a lot of crew died in maints due of spacing, a lot of bodies,  under the shadow, paramed without monitor will not see the others bodies around.

2) People are evil, will just put clothes on dead monkeys to troll parameds to save someone, but instead they spending time to see this trolling.

3) Body could just be moved by shitters from a to b, that also spend paramedic time.

In conclusion parameds will just use crew monitor console as "portable" one, now this would be a bit out of character situation, yikes.

**Comment by Nimfar11** [comment]
It seems to me a dubious decision. Anyone who is afraid of being found simply turns off the sensors. If it gets in the way, just remove the prescription from the technology, as before it will be the only one for the entire station, at the CMO or at the paramedic.

**Comment by Hmeister-real** [comment]
I think this change is too damaging to parameds, their job is already hard enough with most people not having coords on and them not having a mobile crew monitor will make it even harder. If you really want to remove it make it researchable by science or unmap it and put it into parameds locker/bag. Or maybe make a new item that isn't as strong? like maybe a special hud made for parameds which highlights crit people (with coords) with an arrow pointing to them or something.

**Comment by foboscheshir** [comment]
billions must rot :trollface:

**Comment by Holinka4ever** [comment]
Also, my next sentence will be offtopic, but in another way it will be related on topic.

Powergaming and powegamers are not valid argument to these kind prs. I know that no one can teach people to not doing that stuff, but turning game into a "pillow room" because people are doing people stuff out of control, is not a variable answer. Deleting stuff that people over-using it, will not make anything good.

There is no variable option to fix powergaming, only co-exist with it. Cause powergaming is somehow but a dna of the space station.
 
I said my word about it. 

But to stay in topic, so what about to add crew monitor as app in paramed and cmo pdas, hope there will be not a pesky powegamers, whom will beat to death crewmates to get it :trollface:

**Comment by metalgearsloth** [comment]
If you want something handheld pr a significantly nerfed version instead of free portable roundstart maphacks for all non-antags.

**Comment by Ilya246** [comment]
i understand why would you remove the roundstart monitor, but why remove it from the syndicate observation kit, making it pretty much useless, and from biochem tech? biochem is already research-starved and i don't think this is a good decision

**Comment by metalgearsloth** [comment]
> biochem is already research-starved and i don't think this is a good decision

We don't add content for the sake of content as that's just unnecessary tech debt.

> but why remove it from the syndicate observation kit

I'm going to remove the kit.

**Comment by Holinka4ever** [comment]
Now these will bring another type of question:

Why you anyway waited it for 6 hours, if you could just self-merge it from start?
Why you imade it in first place, if you ignored people? 

These is kinda rude behavior imho, cause people brought thoughts and ideas to not just straight deleting it... it feels like that getting betrayed.

**Comment by metalgearsloth** [comment]
> Why you anyway waited it for 6 hours, if you could just self-merge it from start?
> Why you imade it in first place, if you ignored people?

I talked about it on discord with other maintainers. Once talking was done it got merged but more talking can also happen later as that's how development goes.

**Comment by Hmeister-real** [comment]
This is a sad day for all paramedics and rotting people. Can't wait to die in maints, wait 15 minutes and never be found.

**Comment by Aisu9** [comment]
> This is a sad day for all paramedics and rotting people. Can't wait to die in maints, wait 15 minutes and never be found.

me when the medical team have absolute zero social skill and can't keep watch on the stupid monitor and guide me to the soon rotting captain

**Comment by K-Dynamic** [comment]
This change will most definitely be a positive change for parameds, right?

**Comment by Carmell0** [comment]
> We don't add content for the sake of content as that's just unnecessary tech debt.

> Highly disagree. The one thing space station needs the most is more random events and more content
